rsyslog/init.sls

```diff
@@ -50,6 +51,33 @@ rsyslog_mmjsonparse:
     - require:
       - pkg: rsyslog
 
+{%- else %}
+rsyslog_updates:
+  pkgrepo.managed:
+    - name: deb http://ppa.launchpad.net/adiscon/v8-devel/ubuntu focal main
+    - dist: focal
```

Comment:
Only change other than version. rsyslog-mmjsonparse wasn't available in the default Ubuntu 20 ppas.

Reply:
Should we be more specific here? Instead of an else and a resulting "default" state, shouldn't we specify the specific version in which you should be utilize this? I'd rather a case of an unexpected version fail rather than just accept this branch.

Then again, we aren't exactly likely to encounter more than just the two versions we utilize, so I can understand ignoring this criticism as extraneous.

Reply:
Yeah I went with this approach because it was simpler and I didn't have a good default behavior. It should throw an error if it's not Ubuntu 16.04 or Ubuntu 20.04 but there isn't really a good way to throw an error in Salt. The best I found was to just add a line that's verbatim "Error: unsupported Ubuntu Version". This causes a syntax error but it wasn't super clear either. Since Ubuntu 20.04 is the new version I made that the "default" and Ubuntu 16 is the special case. It should make it slightly easier to remove Ubuntu 16.04 down the line.